uaclient/config.py

```diff
@@ -48,6 +50,7 @@ class UAConfig:
         "instance-id": DataPath("instance-id", True),
         "machine-id": DataPath("machine-id", True),
         "machine-token": DataPath("machine-token.json", True),
+        "lock": DataPath("lock", False),
```

Comment:
Why is the lock file not marked as private ?

Reply:
Originally I was trying to write is as non-root user too and update status based on if other things held the lock. I backed out that change as I realized that meant we had to open up world writeable perms on the directory /var/lib/ubuntu-advantage, so I backed out the non-root user manipulating this. I forgot to back out this change when I reverted that code. dropping that now